### DISCUSSION TITLE
.pem key

### DISCUSSION THREAD

**Discussion by robyn-starre** [ScratchAddons/ScratchAddons#2369] [Help] [answered]
Hiya, I am looking for the path of the .pem key of the extension so that I can pack le extension and move it between accounts where it isn't available on the appstore. If you can't help me with this I completely understand but help would be appreciated!

**Comment by cobaltt7** [comment]
There isn't one.

**Reply by cobaltt7** [reply]
nvm, we do

**Comment by easrng** [comment, marked as answer]
The pem key is sensitive and shouldn't be shared. Please use your own key, chrom(ium) generates one when you use the "Pack Extension" tool.

**Reply by robyn-starre** [reply]
I tried to use the "Pack Extension" option but it told me to use the already existing .pem key in order to even _pack_ the extension, so I'm... kinda stuck

**Reply by Sushii64** [reply]
Solution: Use the command 

`chromium --pack-extension=[directory] --pack-extension-key=[directory].pem`

It will make a key for you.

**Reply by Sushii64** [reply]
(let me know if I have to delete this)
